en/flight_controller/omnibus_f4_sd.md

```diff
@@ -171,12 +171,27 @@ Instructions for this are provided in the [TBS Crossfire Manual](https://www.tea
 
 The schematics are provided by [Airbot](https://myairbot.com/): [OmnibusF4-Pro-Sch.pdf](http://bit.ly/obf4pro).
 
-## Build & Upload Instructions
+## Build & Upload Instructions {#upload}
 
 The board comes pre-installed with [Betaflight](https://github.com/betaflight/betaflight/wiki). 
-Before PX4 firmware can be installed, the PX4 bootloader needs to be built and uploaded.
+Before the PX4 firmware can be installed, the PX4 bootloader needs to be flashed.
 
-### Download
+There are two options for flashing the bootloader: via *Betaflight Configurator* (easier), or building from source (guaranteed to work).
+
+### Setup PX4 using Betaflight Configurator {#betaflight_configurator}
+
+To setup PX4 using the *Betaflight Configurator*:
+1. Download the pre-built bootloader binary: [omnibusf4sd_bl.hex](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/omnibus_f4_sd/omnibusf4sd_bl_d52b70cb39.hex).
+1. Download the [Betaflight Configurator](https://github.com/betaflight/betaflight-configurator/releases) for your platform.
```

Comment:
If you have Chrome installed, you can just add the app from https://chrome.google.com/webstore/detail/betaflight-configurator/kdaghagfopacdngbohiknlhcocjccjao.

Reply:
Ah, I misread that Chrome were removing support for extensions - they are just removing inline installation. I'll add both links.